src/libraries/System.Text.RegularExpressions/tests/Regex.Match.Tests.cs

```diff
@@ -1139,5 +1139,15 @@ public void Synchronized()
 
             AssertExtensions.Throws<ArgumentNullException>("inner", () => System.Text.RegularExpressions.Match.Synchronized(null));
         }
+
+        [Fact]
+        public void Explore()
+        {
+            //var regex2 = new Regex("abc(efg|hij)xyz");
+            //var regex1 = new Regex("((abc|def)mno|(xyz|abc)ghi)rst"); // AC word list still has a bug
```

Comment:
This prototype has a bug while populating the word list for this pattern. When walking the tree in `RegexWriter`, there is an error in expanding the word list. I'll fix it later :)